scripts/setup/postgres-init-db

```diff
@@ -2,32 +2,74 @@
 set -e
 set -x
 
+# A bit of a helper variable, default is voyager
+DEPLOYMENT_TYPE="${DEPLOYMENT_TYPE:-voyager}"
+# What user should we use for connecting to the database
+POSTGRES_USER="${POSTGRES_USER:-postgres}"
+
 # Shut down all services to ensure a quiescent state.
-supervisorctl stop all
+# Stop supervisord only if the socket exists, prevents problems with docker
+if [ -e "/var/run/supervisor.sock" ]; then
+    supervisorctl stop all
+fi
```

Comment:
Can you explain the problem with docker this causes?  Is it just supervisorctl throwing an error if it wasn't already running?